SimpleCom/SimpleCom.cpp

```diff
@@ -16,9 +16,9 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
  * 02110-1301, USA.
  */
-#define _CRT_SECURE_NO_WARNINGS
+#include "stdafx.h"
 
-#include <Windows.h>
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
I don't want to add it because the warnings (includes which happens in future) helps us to be aware the bugs.